tests/acceptance/features/apiGraph/changeOwnPassword.feature

```diff
@@ -1,7 +1,6 @@
 @api
 Feature: an user changes its own password
 
-
```

Comment:
This blank space was an intentional change in recent PR, referring to documentation https://github.com/owncloud/docs-server/pull/697/files#diff-12a0f404a5bbbda65e50018a487cd725797106d670f4308b921d2081dc4b3516R1129-R1154:~:text=more%20relevant%20discussion.-,%3D%3D%3D%3D%20Inter%2Dscenario%20spacings,%2D%2D%2D%2D,-Commenting%20on%20lines